docker/xgb/ubuntu/common/sbin/install_packages

```diff
@@ -11,5 +11,10 @@ add-apt-repository -y ppa:deadsnakes/ppa
 apt-get update
 apt-get install -y \
     wget gcc python3.5-dev git python-pip python-virtualenv \
-    libmysqlclient-dev unzip tar \
-    nodejs npm libkrb5-dev
+    mariadb-client \
```

Comment:
just curious, why are we moving towards mariadb?

Reply:
because its impossible to isntall libmysqlclient-dev and node at the same time in ubuntu